onedal/linear_model/tests/test_linear_regression.py

```diff
@@ -53,76 +56,86 @@ def test_pickle(queue):
 
         assert isinstance(model2, model.__class__)
         result = model2.predict(X, queue=queue)
-        assert_array_equal(expected, result)
+
+        assert_allclose(expected, result, rtol=1e-5)

```

Comment:
`rtol=1e-5` for original and pickled model is confusing. Why it should have this value?

Reply:
Done